### PR TITLE
Async operation for instance resize/delete.

### PR DESCRIPTION
**Pull request by sacharya**

### CODE REVIEW

**Comment by cp16net** [comment]
+1

**Comment by imsplitbit** [comment]
+1

File: etc/reddwarf/reddwarf-taskmanager.conf.sample [hunk 1]
```diff
@@ -47,6 +47,19 @@ reddwarf_auth_url = http://0.0.0.0:5000/v2.0
...
+#dns_instance_entry_factory=reddwarf.dns.rsdns.driver.RsDnsInstanceEntryFactory
+#dns_account_id=0
+#dns_domain_id=0
+#dns_username=username
+#dns_passkey=passkey
+#dns_domain_name=test.domain-test.com
+#dns_management_base_url=
+#dns_auth_url=
+#dns_ttl=300
+
```
Comment (hub-cap) [inline etc/reddwarf/reddwarf-taskmanager.conf.sample]: can we just remove these?

**Comment by hub-cap** [comment]
much cleaner +1

**Comment by TimSimpsonR** [comment]
+1

File: reddwarf/taskmanager/manager.py [hunk 2]
```diff
@@ -48,10 +48,23 @@ def _wrapper(self, method, context, *args, **kwargs):
             func = getattr(self, method)
             func(context, *args, **kwargs)
         except Exception as e:
-            excutils.save_and_reraise_exception()
+            LOG.error("Got an error running %s!" % method)
+            LOG.debug(traceback.format_exc())
```
Comment (rnirmal) [inline reddwarf/taskmanager/manager.py]: You should probably make the traceback as an error or exception too

**Comment by rnirmal** [comment]
+1

**Pull request merged**